docker/docker-compose.yaml

```
@@ -0,0 +1,92 @@
+version: '3.8'
+
+# The services below should only be used for development and testing purposes. They are not production-ready.
+services:
+  jenkins-master:
+    build:
+      context: ./docker/master-node
```

```suggestion
      context: ./master-node
```

I think the paths need to be relative to the `docker-compose.yaml` file.

---

It's relative to the build context, which by default is the current folder (the one where the build command is executed).

The command given in the docs is expected to be run from the repo root:
> run `docker-compose -p datadog-jenkins-plugin -f docker/docker-compose.yaml up` from **the directory where this repository is cloned**

---

Indeed it fails with the newer versions of `docker-compose`. Fixed the paths and added a note to the doc saying that `docker-compose` needs to be updated if there are "path ... not found" errors.